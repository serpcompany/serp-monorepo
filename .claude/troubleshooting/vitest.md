https://vitest.dev/guide/common-errors.html