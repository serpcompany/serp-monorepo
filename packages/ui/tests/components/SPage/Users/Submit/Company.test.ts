/* eslint-disable @typescript-eslint/no-unused-vars, no-unused-vars  */

import { mockNuxtImport } from '@nuxt/test-utils/runtime';
import { describe, expect, it, vi } from 'vitest';
import { ref } from 'vue';
import SubmitCompany from '../../../../../components/SPage/Users/Submit/Company.vue';
import ComponentRender from '../../../../componentRender';

// Global mock for useRoute
let routeMock = { query: {} };
globalThis.useRoute = () => routeMock;

// Global mock for useUserSession
let session_;
globalThis.useUserSession = () => session_;

// Global mock for S3 dependency
globalThis.useS3Object = () => ({
  upload: async () => '/api/s3/query/dummy'
});

// Global mocks for useCompanyCategories and useCompanySubmissions
globalThis.useCompanyCategories = async () => [
  { id: 1, name: 'Tech', slug: 'tech' },
  { id: 2, name: 'Finance', slug: 'finance' }
];

globalThis.useCompanySubmissions = async (id: string) => {
  return {
    id: 123,
    approved: false,
    name: 'Test Company',
    domain: 'testcompany.com',
    pricing: 'Paid',
    tags: 'innovative, startup',
    oneLiner: 'Best company ever',
    description: 'Detailed description',
    categories: [1, 2],
    logo: 'logo.png',
    uuid: 'uuid-123',
    isPriority: false
  };
};

// Other dependency mocks via mockNuxtImport
mockNuxtImport('useToast', () => () => ({
  add: vi.fn()
}));

mockNuxtImport('useRuntimeConfig', () => () => ({
  public: {
    cloudflareR2PublicUrl: 'https://dummy.cloudflare/'
  }
}));

describe('SPage/Users/Submit/Company Snapshot', () => {
  const scenarios: [
    string,
    { session: unknown; route: Record<string, unknown> }
  ][] = [
    [
      'New Submission (no id)',
      {
        session: {
          loggedIn: ref(true),
          user: ref({ email: 'test@test.com' })
        },
        route: { query: {} }
      }
    ],
    [
      'Existing Submission (with id)',
      {
        session: {
          loggedIn: ref(true),
          user: ref({ email: 'test@test.com' })
        },
        route: { query: { id: 'existing' } }
      }
    ]
  ];

  it.each(scenarios)(
    'renders %s correctly',
    async (desc, { session, route }) => {
      // Update the mutable session and route for the current scenario
      session_ = session;
      routeMock = route;

      const html = await ComponentRender(
        `SPage/Users/Submit/Company ${desc}`,
        {},
        SubmitCompany
      );
      expect(html).toMatchSnapshot();
    }
  );
});
