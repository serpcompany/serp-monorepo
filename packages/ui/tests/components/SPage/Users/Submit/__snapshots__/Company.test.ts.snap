// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SPage/Users/Submit/Company Snapshot > renders Existing Submission (with id) correctly 1`] = `
"<div data-v-8dd619f3="" class="flex flex-col lg:grid lg:grid-cols-10 lg:gap-10">
  <!--v-if-->
  <div class="lg:col-span-10">
    <main data-v-8dd619f3="" class="min-h-[calc(100vh-var(--ui-header-height))]">
      <div data-v-8dd619f3="" class="rounded-[calc(var(--ui-radius)*2)] bg-(--ui-bg) ring ring-(--ui-border) divide-y divide-(--ui-border)">
        <!--v-if-->
        <div class="p-4 sm:p-6">
          <form data-v-8dd619f3="">
            <div data-v-8dd619f3="" class="text-sm w-full">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-0" class="block font-medium text-(--ui-text) after:content-['*'] after:ms-0.5 after:text-(--ui-error)">Company Name</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <div data-v-8dd619f3="" class="relative inline-flex items-center w-full"><input id="v-0-0-0" type="text" placeholder="SERP AI" class="w-full rounded-[calc(var(--ui-radius)*1.5)] border-0 placeholder:text-(--ui-text-dimmed) focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-(--ui-text-highlighted) bg-(--ui-bg) ring ring-inset ring-(--ui-border-accented) focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-(--ui-primary)" autocomplete="off" aria-describedby="" aria-invalid="false" value="">
                  <!--v-if-->
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm mt-4 w-full">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-1" class="block font-medium text-(--ui-text) after:content-['*'] after:ms-0.5 after:text-(--ui-error)">Domain</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <div data-v-8dd619f3="" class="relative inline-flex items-center w-full"><input id="v-0-0-1" type="text" placeholder="serp.ai" class="w-full rounded-[calc(var(--ui-radius)*1.5)] border-0 placeholder:text-(--ui-text-dimmed) focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-(--ui-text-highlighted) bg-(--ui-bg) ring ring-inset ring-(--ui-border-accented) focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-(--ui-primary)" autocomplete="off" aria-describedby="" aria-invalid="false" value="">
                  <!--v-if-->
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm mt-4 w-full">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-2" class="block font-medium text-(--ui-text) after:content-['*'] after:ms-0.5 after:text-(--ui-error)">Pricing</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <!---->
                <div dir="ltr" id="v-0-0-2" class="relative inline-flex items-center w-full"><input aria-disabled="false" type="text" aria-expanded="false" aria-controls="" aria-autocomplete="list" role="combobox" autocomplete="false" aria-describedby="" aria-invalid="false" class="rounded-[calc(var(--ui-radius)*1.5)] transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-(--ui-text-highlighted) bg-(--ui-bg) ring ring-inset ring-(--ui-border-accented) w-full border-0 placeholder:text-(--ui-text-dimmed) focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-(--ui-primary) pe-9" value="">
                  <!--v-if--><button type="button" tabindex="-1" aria-label="Show popup" aria-haspopup="listbox" aria-expanded="false" aria-controls="" data-state="closed" aria-disabled="false" class="group absolute inset-y-0 end-0 flex items-center disabled:cursor-not-allowed disabled:opacity-75 pe-2.5"><span class="iconify i-lucide:chevron-down shrink-0 text-(--ui-text-dimmed) size-5" aria-hidden="true"></span></button>
                  <!--teleport start-->
                  <!--teleport end-->
                  <!---->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm mt-4 w-full">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-3" class="block font-medium text-(--ui-text)">Category(s)</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <!---->
                <div dir="ltr" id="v-0-0-3" class="relative inline-flex items-center flex-wrap w-full rounded-[calc(var(--ui-radius)*1.5)] transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-(--ui-text-highlighted) bg-(--ui-bg) ring ring-inset ring-(--ui-border-accented) has-focus-visible:ring-2 has-focus-visible:ring-(--ui-primary) focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-(--ui-primary) pe-9"><input type="text" autocomplete="false" autocorrect="off" autocapitalize="off" aria-describedby="" aria-invalid="false" class="border-0 bg-transparent placeholder:text-(--ui-text-dimmed) focus:outline-none disabled:cursor-not-allowed disabled:opacity-75" aria-disabled="false" aria-expanded="false" aria-controls="" aria-autocomplete="list" role="combobox" dir="ltr" value="">
                  <!---->
                  <!--v-if--><button type="button" tabindex="-1" aria-label="Show popup" aria-haspopup="listbox" aria-expanded="false" aria-controls="" data-state="closed" aria-disabled="false" class="group absolute inset-y-0 end-0 flex items-center disabled:cursor-not-allowed disabled:opacity-75 pe-2.5"><span class="iconify i-lucide:chevron-down shrink-0 text-(--ui-text-dimmed) size-5" aria-hidden="true"></span></button>
                </div>
                <!--teleport start-->
                <!--teleport end-->
                <!---->
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm mt-4 w-full">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-4" class="block font-medium text-(--ui-text)">Tags</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <div data-v-8dd619f3="" class="relative inline-flex items-center w-full"><input id="v-0-0-4" type="text" placeholder="e.g. AI, SEO, Marketing" class="w-full rounded-[calc(var(--ui-radius)*1.5)] border-0 placeholder:text-(--ui-text-dimmed) focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-(--ui-text-highlighted) bg-(--ui-bg) ring ring-inset ring-(--ui-border-accented) focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-(--ui-primary)" autocomplete="off" aria-describedby="" aria-invalid="false" value="">
                  <!--v-if-->
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm mt-8 w-full">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-5" class="block font-medium text-(--ui-text)">Company Logo</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative"><input data-v-8dd619f3="" type="file" accept="image/*">
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm mt-8 w-full">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-6" class="block font-medium text-(--ui-text) after:content-['*'] after:ms-0.5 after:text-(--ui-error)">One-Liner</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <div data-v-8dd619f3="" class="relative inline-flex items-center w-full"><input id="v-0-0-6" type="text" placeholder="A short company tagline (max 75 characters)" class="w-full rounded-[calc(var(--ui-radius)*1.5)] border-0 placeholder:text-(--ui-text-dimmed) focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-(--ui-text-highlighted) bg-(--ui-bg) ring ring-inset ring-(--ui-border-accented) focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-(--ui-primary)" autocomplete="off" maxlength="75" aria-describedby="" aria-invalid="false" value="">
                  <!--v-if-->
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="col-span-2">
              <div data-v-8dd619f3="" class="text-sm mt-4 w-full">
                <div class="">
                  <div class="flex content-center items-center justify-between"><label for="v-0-0-7" class="block font-medium text-(--ui-text) after:content-['*'] after:ms-0.5 after:text-(--ui-error)">Description</label>
                    <!--v-if-->
                  </div>
                  <!--v-if-->
                </div>
                <div class="mt-1 relative">
                  <div data-v-8dd619f3="" class="relative inline-flex items-center w-full"><textarea id="v-0-0-7" rows="5" placeholder="A detailed overview of your company" class="w-full rounded-none border-0 placeholder:text-[var(--ui-text-dimmed)] focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-(--ui-text-highlighted) bg-(--ui-bg) ring ring-inset ring-(--ui-border-accented) focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-(--ui-primary)" aria-describedby="" aria-invalid="false" value=""></textarea></div>
                  <!--v-if-->
                </div>
              </div>
            </div><button type="button" disabled="" class="rounded-none font-medium inline-flex items-center focus:outline-hidden disabled:cursor-not-allowed aria-disabled:cursor-not-allowed disabled:opacity-75 aria-disabled:opacity-75 transition-colors px-3 py-2 text-base gap-2 text-(--ui-bg) bg-(--ui-secondary) hover:bg-(--ui-secondary)/75 disabled:bg-(--ui-secondary) aria-disabled:bg-(--ui-secondary) focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-(--ui-secondary) mt-4">
              <!--v-if-->Submit
              <!--v-if-->
            </button>
          </form>
        </div>
        <!--v-if-->
      </div>
    </main>
  </div>
  <!--v-if-->
</div>"
`;

exports[`SPage/Users/Submit/Company Snapshot > renders New Submission (no id) correctly 1`] = `
"<div data-v-8dd619f3="" class="flex flex-col lg:grid lg:grid-cols-10 lg:gap-10">
  <!--v-if-->
  <div class="lg:col-span-10">
    <main data-v-8dd619f3="" class="min-h-[calc(100vh-var(--ui-header-height))]">
      <div data-v-8dd619f3="" class="rounded-[calc(var(--ui-radius)*2)] bg-(--ui-bg) ring ring-(--ui-border) divide-y divide-(--ui-border)">
        <!--v-if-->
        <div class="p-4 sm:p-6">
          <form data-v-8dd619f3="">
            <div data-v-8dd619f3="" class="text-sm w-full">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-0" class="block font-medium text-(--ui-text) after:content-['*'] after:ms-0.5 after:text-(--ui-error)">Company Name</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <div data-v-8dd619f3="" class="relative inline-flex items-center w-full"><input id="v-0-0-0" type="text" placeholder="SERP AI" class="w-full rounded-[calc(var(--ui-radius)*1.5)] border-0 placeholder:text-(--ui-text-dimmed) focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-(--ui-text-highlighted) bg-(--ui-bg) ring ring-inset ring-(--ui-border-accented) focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-(--ui-primary)" autocomplete="off" aria-describedby="" aria-invalid="false" value="">
                  <!--v-if-->
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm mt-4 w-full">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-1" class="block font-medium text-(--ui-text) after:content-['*'] after:ms-0.5 after:text-(--ui-error)">Domain</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <div data-v-8dd619f3="" class="relative inline-flex items-center w-full"><input id="v-0-0-1" type="text" placeholder="serp.ai" class="w-full rounded-[calc(var(--ui-radius)*1.5)] border-0 placeholder:text-(--ui-text-dimmed) focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-(--ui-text-highlighted) bg-(--ui-bg) ring ring-inset ring-(--ui-border-accented) focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-(--ui-primary)" autocomplete="off" aria-describedby="" aria-invalid="false" value="">
                  <!--v-if-->
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm mt-4 w-full">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-2" class="block font-medium text-(--ui-text) after:content-['*'] after:ms-0.5 after:text-(--ui-error)">Pricing</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <!---->
                <div dir="ltr" id="v-0-0-2" class="relative inline-flex items-center w-full"><input aria-disabled="false" type="text" aria-expanded="false" aria-controls="" aria-autocomplete="list" role="combobox" autocomplete="false" aria-describedby="" aria-invalid="false" class="rounded-[calc(var(--ui-radius)*1.5)] transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-(--ui-text-highlighted) bg-(--ui-bg) ring ring-inset ring-(--ui-border-accented) w-full border-0 placeholder:text-(--ui-text-dimmed) focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-(--ui-primary) pe-9" value="">
                  <!--v-if--><button type="button" tabindex="-1" aria-label="Show popup" aria-haspopup="listbox" aria-expanded="false" aria-controls="" data-state="closed" aria-disabled="false" class="group absolute inset-y-0 end-0 flex items-center disabled:cursor-not-allowed disabled:opacity-75 pe-2.5"><span class="iconify i-lucide:chevron-down shrink-0 text-(--ui-text-dimmed) size-5" aria-hidden="true"></span></button>
                  <!--teleport start-->
                  <!--teleport end-->
                  <!---->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm mt-4 w-full">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-3" class="block font-medium text-(--ui-text)">Category(s)</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <!---->
                <div dir="ltr" id="v-0-0-3" class="relative inline-flex items-center flex-wrap w-full rounded-[calc(var(--ui-radius)*1.5)] transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-(--ui-text-highlighted) bg-(--ui-bg) ring ring-inset ring-(--ui-border-accented) has-focus-visible:ring-2 has-focus-visible:ring-(--ui-primary) focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-(--ui-primary) pe-9"><input type="text" autocomplete="false" autocorrect="off" autocapitalize="off" aria-describedby="" aria-invalid="false" class="border-0 bg-transparent placeholder:text-(--ui-text-dimmed) focus:outline-none disabled:cursor-not-allowed disabled:opacity-75" aria-disabled="false" aria-expanded="false" aria-controls="" aria-autocomplete="list" role="combobox" dir="ltr" value="">
                  <!---->
                  <!--v-if--><button type="button" tabindex="-1" aria-label="Show popup" aria-haspopup="listbox" aria-expanded="false" aria-controls="" data-state="closed" aria-disabled="false" class="group absolute inset-y-0 end-0 flex items-center disabled:cursor-not-allowed disabled:opacity-75 pe-2.5"><span class="iconify i-lucide:chevron-down shrink-0 text-(--ui-text-dimmed) size-5" aria-hidden="true"></span></button>
                </div>
                <!--teleport start-->
                <!--teleport end-->
                <!---->
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm mt-4 w-full">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-4" class="block font-medium text-(--ui-text)">Tags</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <div data-v-8dd619f3="" class="relative inline-flex items-center w-full"><input id="v-0-0-4" type="text" placeholder="e.g. AI, SEO, Marketing" class="w-full rounded-[calc(var(--ui-radius)*1.5)] border-0 placeholder:text-(--ui-text-dimmed) focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-(--ui-text-highlighted) bg-(--ui-bg) ring ring-inset ring-(--ui-border-accented) focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-(--ui-primary)" autocomplete="off" aria-describedby="" aria-invalid="false" value="">
                  <!--v-if-->
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm mt-8 w-full">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-5" class="block font-medium text-(--ui-text)">Company Logo</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative"><input data-v-8dd619f3="" type="file" accept="image/*">
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="text-sm mt-8 w-full">
              <div class="">
                <div class="flex content-center items-center justify-between"><label for="v-0-0-6" class="block font-medium text-(--ui-text) after:content-['*'] after:ms-0.5 after:text-(--ui-error)">One-Liner</label>
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
              <div class="mt-1 relative">
                <div data-v-8dd619f3="" class="relative inline-flex items-center w-full"><input id="v-0-0-6" type="text" placeholder="A short company tagline (max 75 characters)" class="w-full rounded-[calc(var(--ui-radius)*1.5)] border-0 placeholder:text-(--ui-text-dimmed) focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-(--ui-text-highlighted) bg-(--ui-bg) ring ring-inset ring-(--ui-border-accented) focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-(--ui-primary)" autocomplete="off" maxlength="75" aria-describedby="" aria-invalid="false" value="">
                  <!--v-if-->
                  <!--v-if-->
                </div>
                <!--v-if-->
              </div>
            </div>
            <div data-v-8dd619f3="" class="col-span-2">
              <div data-v-8dd619f3="" class="text-sm mt-4 w-full">
                <div class="">
                  <div class="flex content-center items-center justify-between"><label for="v-0-0-7" class="block font-medium text-(--ui-text) after:content-['*'] after:ms-0.5 after:text-(--ui-error)">Description</label>
                    <!--v-if-->
                  </div>
                  <!--v-if-->
                </div>
                <div class="mt-1 relative">
                  <div data-v-8dd619f3="" class="relative inline-flex items-center w-full"><textarea id="v-0-0-7" rows="5" placeholder="A detailed overview of your company" class="w-full rounded-none border-0 placeholder:text-[var(--ui-text-dimmed)] focus:outline-none disabled:cursor-not-allowed disabled:opacity-75 transition-colors px-2.5 py-1.5 text-sm gap-1.5 text-(--ui-text-highlighted) bg-(--ui-bg) ring ring-inset ring-(--ui-border-accented) focus-visible:ring-2 focus-visible:ring-inset focus-visible:ring-(--ui-primary)" aria-describedby="" aria-invalid="false" value=""></textarea></div>
                  <!--v-if-->
                </div>
              </div>
            </div><button type="button" disabled="" class="rounded-none font-medium inline-flex items-center focus:outline-hidden disabled:cursor-not-allowed aria-disabled:cursor-not-allowed disabled:opacity-75 aria-disabled:opacity-75 transition-colors px-3 py-2 text-base gap-2 text-(--ui-bg) bg-(--ui-secondary) hover:bg-(--ui-secondary)/75 disabled:bg-(--ui-secondary) aria-disabled:bg-(--ui-secondary) focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-(--ui-secondary) mt-4">
              <!--v-if-->Submit
              <!--v-if-->
            </button>
          </form>
        </div>
        <!--v-if-->
      </div>
    </main>
  </div>
  <!--v-if-->
</div>"
`;
